docs/source/index.rst

```diff
@@ -5,33 +5,35 @@ Tao of workflows
-With the focus being on the JSON object produced means that the library and client could be
-replaced independently as required by the use case. The reason for this is to be technology
-independent. The server should not care about the client and the client should not care about the
-server (baring interfaces).
+**Workflows are a JSON object encoding actions to be taken by the client.**
+With the focus being on the JSON object produced, the library and client could be
+replaced independently as required by the use case. This is motivated by the idea of being technologically
+independent, that is to say that a client or server which obeys the interfaces and patterns explained in these docs
+will function irrespective of technology used to implement them; they are technologically agnostic.
+The server should not care about the client, and in turn the client should not care about the
+server (barring of course, the interfaces).
 
 Workflows features should be written with some key principles in mind:
 
-* **The client's job is to display information.** Processing and domain logic should be handled by the backend (note: handled might mean defining a task in the workflow).
+* **The job of the client is to display information.** Processing and domain logic should be handled by the backend (note: handled might mean defining a task in the workflow).
 
-* **Ease of execution of the JSON object should be top priority.** This means where possible complexity should be in the library producing the workflow. As it can be hidden from the user of the library by the tooling within the library.
+* **Ease of execution of the JSON object should be top priority.** This means where possible complexity should be in the library producing the workflow, rather than in the client. This may produce complex JSON, but this is obfuscated from the user of the library by tooling within it.
 
-* **Minimize surprise.** We should try to make things as least surprising as possible if there is a direct comparison with something in a programing language it should act the same.
+* **Minimize surprise.** We should try to make things the least surprising as possible: if there is a direct comparison with something in a programing language it should act the same.
 
-* **Workflow primitives should be kept to a minimum.** It needs fit in someones head to be understandable. This is another place there tooling is key if you can construct a feature out of primitives do it. Then hide the construction behind tooling. This allows the things required to implement in the client to be minimal.
+* **Workflow primitives should be kept to a minimum.** It needs to fit in a developers head to be understandable and have utility. This is another place there tooling is key; if you can construct a feature from primitives, do. Hide this construction behind tooling. This allows the things requiring implementation in the client to be minimal.
```

Comment:
hmmmmmmmmmmmmmmmmmmmmmmmm the last sentence still doesn't read right

Reply:
This allows the things the client is required to implement in the client to be minimal